### DISCUSSION TITLE
Use Chronos Config to set date/time

### DISCUSSION THREAD

**Discussion by ErwinAnwyhere365** [jensrossbach/node-red-contrib-chronos#81] [Ideas & Requests]
Hi Jens,

As discussed in on our other topic I would like to raise a feature request to use the chronos config to set the date time in the node itself. In this way you will we able to use the configured chronos locations date time in stead of the Node Red Instance time.

Kind regards,
Erwin

**Comment by jensrossbach** [comment]
To be honest, with "feature request" I meant an [issue](https://github.com/jensrossbach/node-red-contrib-chronos/issues) with that type. Would it be okay for you to create the issue?

**Reply by jensrossbach** [reply]
Feature has been implemented with release v1.17.0.
